internal/constant/constant.go

```diff
@@ -18,4 +18,6 @@ var (
 	SeaweedFsChunkSize      = 1048576 // 1 MiB
 
 	MaxInvitationCodeChar = 6
+
+	DefaultProfileUrl = "/workspaces/1/profile"
```

Comment:
Can we use `/workspaces/default/profile` instead?

Reply:
I am afraid `fiber`'s `ParamsParser` doesn't allow that. Is there any workaround? 

Can we perhaps make a route for specifically getting default assets/files /pictures? Isn't that a tad bit too unnecessary?

![Screenshot 2567-02-21 at 09 34 33](https://github.com/codern-org/codern/assets/68505570/10f15dd7-4c42-4d5a-ae1d-b21d8c5f0ac0)

Reply:
I tried using id 0 as you requested too. But to no avail. It seem like `validator.Struct` in `internal/validator/payload.go` wouldn't count payload with id 0 as `valid payload`